mpa/det/stage.py

```diff
@@ -124,10 +124,14 @@ def configure_data(self, cfg, training, **kwargs):
                 )
             if 'dataset' in cfg.data.train:
                 train_cfg = self.get_train_data_cfg(cfg)
-                train_cfg.otx_dataset = cfg.data.train.pop('otx_dataset', None)
-                train_cfg.labels = cfg.data.train.get('labels', None)
-                train_cfg.data_classes = cfg.data.train.pop('data_classes', None)
-                train_cfg.new_classes = cfg.data.train.pop('new_classes', None)
+                if cfg.data.train.get('otx_dataset', None) is not None:
+                    train_cfg.otx_dataset = cfg.data.train.pop('otx_dataset')
+                if cfg.data.train.get('labels', None) is not None:
+                    train_cfg.labels = cfg.data.train.get('labels')
+                if cfg.data.train.get('data_classes', None) is not None:
+                    train_cfg.data_classes = cfg.data.train.pop('data_classes')
+                if cfg.data.train.get('new_classes', None) is not None:
+                    train_cfg.new_classes = cfg.data.train.pop('new_classes')
```

Comment:
Any issues found in the previous code?

Reply:
Why do we need this block of code in the first place? Do we have any use case?
In my opinion, `otx_dataset`, etc. in `cfg.data.train`, which is a wrapper dataset, have to be considered a bug.
I strongly encouraged to get rid of it.

Reply:
Yeah, I agree. `otx_dataset`, which is actually not a config object but a real object, is kind of hack to feed Geti dataset to mmcv frameworks. If-else logics here need to be refined anyway.
Let's do it it separate PRs.